lib/routes/mixi2/community.ts

```diff
@@ -0,0 +1,70 @@
+import { type Data, type Route, ViewType } from '@/types';
+import type { Context } from 'hono';
+import { getClient, parsePost } from './utils';
+import { parseDate } from '@/utils/parse-date';
+
+const handler = async (ctx: Context) => {
+    const limit = Number.parseInt(ctx.req.query('limit') ?? '20', 10);
+    const communityId = ctx.req.param('id');
+    const mediaOnly = ctx.req.param('media') === 'media';
+
+    const client = getClient();
+
+    const [communityInfo, postsData] = await Promise.all([
+        client.getCommunity({
+            communityId,
+        }),
+        client.getCommunityTimeline({
+            communityId,
+            limit,
+            mediaOnly,
+        }),
+    ]);
+
+    return {
+        title: `${communityInfo.community.name} - ${mediaOnly ? 'メディア' : 'ポスト'}`,
+        image: communityInfo.community.coverImage.postImage?.largeImageUrl,
+        item:
+            postsData?.posts
+                ?.filter((post) => !post.isDeleted)
+                .map((post) => ({
+                    title: communityInfo.community.name,
+                    description: parsePost(post),
+                    pubDate: parseDate(post.createdAt.seconds * 1e3),
+                    guid: post.postId,
+                })) ?? [],
```

Comment:
You can use `getPersonas` which provides user handles and ultimately provide the post link `https://mixi.social/@handle/posts/${post.postId}` and author